tests/pytests/test_data_naif.py

```diff
@@ -1,54 +1,133 @@
+
+    def test_focal2pixel_samples(self):
+        with patch('ale.base.label_pds3.Pds3Label.instrument_id', new_callable=PropertyMock) as instrument_id:
+            instrument_id.return_value = 'MRO_CTX'
+            np.testing.assert_array_equal(self.driver.focal2pixel_samples, [0.0, 0.0, 142.85714285714])
+
+    def test_pixel2focal_x(self):
+        with patch('ale.base.label_pds3.Pds3Label.instrument_id', new_callable=PropertyMock) as instrument_id:
+            instrument_id.return_value = 'MRO_CTX'
+            np.testing.assert_array_equal(self.driver.pixel2focal_x, [0.0, 0.0, 0.007])
+
+    def test_pixel2focal_y(self):
+        with patch('ale.base.label_pds3.Pds3Label.instrument_id', new_callable=PropertyMock) as instrument_id:
+            instrument_id.return_value = 'MRO_CTX'
+            np.testing.assert_array_equal(self.driver.pixel2focal_y, [0.0, 0.007, 0.0])
+
+    def test_focal_length(self):
+        with patch('ale.base.label_pds3.Pds3Label.instrument_id', new_callable=PropertyMock) as instrument_id:
+            instrument_id.return_value = 'MRO_CTX'
+            assert self.driver.focal_length == 352.9271664
+
+    def test_pixel_size(self):
+        with patch('ale.base.label_pds3.Pds3Label.instrument_id', new_callable=PropertyMock) as instrument_id:
+            instrument_id.return_value = 'MRO_CTX'
+            assert self.driver.pixel_size == 7e-06
+
+    def test_target_body_radii(self):
+        with patch('ale.base.label_pds3.Pds3Label.target_name', new_callable=PropertyMock) as target_name:
+            target_name.return_value = 'Mars'
+            np.testing.assert_array_equal(self.driver.target_body_radii, [3396.19, 3396.19, 3376.2 ])
+
+    def test_reference_frame(self):
+        with patch('ale.base.label_pds3.Pds3Label.target_name', new_callable=PropertyMock) as target_name:
+            target_name.return_value = 'Mars'
+            assert self.driver.reference_frame == 'IAU_Mars'
+
+    def test_ephemeris_start_time(self):
+        with patch('ale.base.label_pds3.Pds3Label.spacecraft_name', new_callable=PropertyMock) as spacecraft_name, \
+             patch('ale.base.label_pds3.Pds3Label.spacecraft_clock_start_count', new_callable=PropertyMock) as spacecraft_clock_start_count:
+            spacecraft_name.return_value = 'MRO'
+            spacecraft_clock_start_count.return_value = '0'
+
+            assert self.driver.ephemeris_start_time == -631195148.8160816
+
+    def test_ephemeris_start_time(self):
+        with patch('ale.base.label_pds3.Pds3Label.spacecraft_name', new_callable=PropertyMock) as spacecraft_name, \
+             patch('ale.base.label_pds3.Pds3Label.spacecraft_clock_stop_count', new_callable=PropertyMock) as spacecraft_clock_stop_count:
+            spacecraft_name.return_value = 'MRO'
+            spacecraft_clock_stop_count.return_value = '1/60000'
+
+            assert self.driver.ephemeris_stop_time == -631135148.8160615
```

Comment:
Why are there 2 tests for this?

Reply:
Miss name, should be a start and stop time test.